src/main/template/main.handlebars

```diff
@@ -11,16 +11,35 @@
                 <div class="scope-selector">
                     <div class="area controls">
                         <label for="input_baseUrl">Base URL</label>
-                        <span id="input_baseUrl" style="word-wrap: break-word;font-size:1.2rem;margin-bottom: 17px;display: block;" >{{baseUrl}}</span>
+                        <div class="input-group" style="margin-bottom: 0.5rem;">
+                            <span id="input_baseUrl">{{baseUrl}}</span>
```

Comment:
Thanks! A lot of this PR is just minor changes like this one. I think the small differences are adding up to good results